packages/themes/scss/basil-green.scss

```diff
@@ -0,0 +1,28 @@
+@import url("https://fonts.googleapis.com/css?family=Pacifico");
+@import url("https://fonts.googleapis.com/css?family=Roboto");
```

Comment:
you should add `&display=swap` to the url (e.g., see the Robot URL in baby-blue)

Reply:
Resolved!